player-events.js

```diff
@@ -157,12 +157,14 @@ function getRoom(state, player, nextRoomId) {
   let nextRoom = null;
   if (state.AreaFactory.isInstanced(nextAreaName)) {
     const instanceId = player.name;
-    B.sayAt(player, `Entering instanced area: ${nextAreaName}`);
+    if (player.area != nextAreaName ){
```

Comment:
Some of the spacing here is a bit off, and it is best practice to use `!==` (though I am not sure this would cause issues unless the area names were undefined/null/`0`/`''`...

Reply:
Whats the best way for me to edit the pull request?  do I just need to submit another one?

Reply:
Nah, you can edit it using GitHub's UI, or you can use Git to pull down the branch, edit it locally, and then push it to the same remote branch.